excel-importer/build.gradle

```diff
@@ -19,4 +19,7 @@ dependencies {
     compile "org.apache.commons:commons-collections4:4.1"
     compile "org.apache.poi:poi-ooxml:3.17"
     testCompile "org.springframework.boot:spring-boot-starter-test"
+    compile group: 'com.microsoft.azure', name: 'azure-storage', version: '7.0.0'
+    testCompile group: 'org.powermock', name: 'powermock-api-mockito', version: '1.7.4'
```

Comment:
Can you check the latest azure-storage version you can use

Reply:
Upgraded to version 8.0.0 (as used in Document Management Store app)